package.json

```diff
@@ -163,6 +163,7 @@
     "@types/use-sync-external-store": "^0.0.3",
     "@typescript-eslint/eslint-plugin": "^6.2.1",
     "@typescript-eslint/parser": "^6.2.1",
+    "@veake/prettier-config": "^1.0.0",
```

Comment:
I thinkning is the incorrect command, but right now I success to running just `yarn add prettier` without this `@veake/prettier-config`

This is the changes that you are mentioned all the time I hope so, is what I have now in my code
Just let me to know if this correct, and I push it
<img width="822" alt="צילום מסך 2023-09-08 132956" src="https://github.com/pmndrs/zustand/assets/114335827/233a05e2-cb23-4960-81d1-60d4f95f9cfd">

Reply:
I'm confused and sorry if things confuse you.
I'd expect this PR to change only one file in docs.

> change just two files as you wrote

Did we say to change package.json and yarn.lock files? Sorry for taking your time.

Reply:
All good, I'm also confused...
So please you can explain to me from the start again what is all the changes that you expect them?
I ran the command `yarn add prettier` as you mentioned, and now it has changed just two files instead of a lot of files as it happened in the last time.

Reply:
This PR is to improve docs.
The target file is `docs/guides/updating-state.md`.
What's expected is to change only that file.
af964ff changes three files compared to `main` branch.

I don't know what's wrong as usually we don't have such a problem.
If you don't have LF issues, I think reverting the last commit and running `yarn` and `yarn run prettier` should change only one file.

Reply:
@OshriAsulin Check if you are running `yarn run prettier`, not `yarn add prettier`.

The `add` command will install new packages, that is **not** what you want here.

Reply:
<img width="944" alt="צילום מסך 2023-09-10 012628" src="https://github.com/pmndrs/zustand/assets/114335827/6566ae39-90f7-4163-896c-de13c256dac4">

@sewera 
As you can see `yarn run prettier` also didn't help because he modified a lot of files, I really don't understand why

Reply:
> This PR is to improve docs. The target file is `docs/guides/updating-state.md`. What's expected is to change only that file. [af964ff](https://github.com/pmndrs/zustand/commit/af964ffb7255bd53ccd040565cba42c3eca18cea) changes three files compared to `main` branch.
> 
> I don't know what's wrong as usually we don't have such a problem. If you don't have LF issues, I think reverting the last commit and running `yarn` and `yarn run prettier` should change only one file.

I fixed all the commits to one commit, but I didn't success in running the prettier command

Reply:
> didn't help because he modified a lot of files

It looks like CRLF/LF difference issue as pointed out.